modules/cnf-image-based-upgrade-shared-container-partition.adoc

```diff
@@ -65,4 +79,12 @@ spec:
 ----
 <1> Specify the root disk.
 <2> Specify the start of the partition in MiB. If the value is too small, the installation will fail.
-<3> Specify a minimum size for the partition of 500 GB to ensure adequate disk space for precached images. If the value is too small, the deployments after installation will fail.
\ No newline at end of file
+<3> Specify a minimum size for the partition of 500 GB to ensure adequate disk space for precached images. If the value is too small, the deployments after installation will fail.
```

Comment:
```suggestion
<3> Specify a minimum size of 500 GB for the partition. This ensures that adequate disk space for precached images exists.
```

Reply:
Leaving as is as it effects IBU content too and there was much back and forth about this.